e2e-tests/development-runtime/cypress/integration/gatsby-preview/updating.js

```diff
@@ -15,17 +15,17 @@ describe(`Gatsby Preview (Updating)`, () => {
   it(`displays initial data`, () => {
     cy.get(`li:eq(0) a`).click().waitForRouteChange()
 
-    cy.queryByText(`Hello World (1)`).should(`exist`)
+    cy.findByText(`Hello World (1)`).should(`exist`)
```

Comment:
That's one of breaking changes in `@testing-library/cypress@6` - https://github.com/testing-library/cypress-testing-library/releases/tag/v6.0.0

This comment applies to all replacements like that here